docs/proposed/http-storage-node-protocol.rst

```diff
@@ -738,7 +738,7 @@ Reading
 ``GET /v1/mutable/:storage_index/shares``
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
-Retrieve a list indicating all shares available for the indicated storage index.
+Retrieve a set indicating all shares available for the indicated storage index.
 For example::
 
   [1, 5]
```

Comment:
This still looks like a list (or an array, I guess).  Since cbor supports sets presumably the value will actually be a set there.  Since json doesn't, if the client demands json I guess they'll still get an array.  Maybe we should say something about that.